### ISSUE TITLE
New SC for keyboard operation?

### ISSUE THREAD

**Issue by JAWS-test** [w3c/wcag3#188]
I propose a new success criterion for WCAG 2.2 as a supplement to 2.1.1:
"If a standard operation exists for navigating through the page and for operating the interactive elements, then this should be implemented. Divergences should be documented in the application. <ins title="Added on 26 August 2019, 4pm">Exception: The operation is controlled by the user agent and is not modified by the author</ins>".

Reason:
* 2.1.1 only requires keyboard operation, but does not specify how this should be done. I.e. according to 2.1.1 it would be ok if the buttons could only be activated via Alt+Shift+X and this is not documented anywhere (see https://github.com/w3c/wcag/issues/857#issuecomment-521319817).
* In 2.1.2 it is specified for keyboard traps that the standard operation must work and that divergences from the standard operation must be documented. 
* If an application implements deviating operating methods, it runs the risk of no longer functioning with the screen reader, since the screen reader does not pass many keystrokes to the browser in reading mode, but uses them for its own functions (this does not apply if role=application is used or ARIA roles that cause the switch to application mode or form mode). However, if a blind user receives an indication of the used operating method in the application, he or she can manually switch the screen reader to application or form mode or use an access key to pass individual key entries directly to the browser.
* Even without a screenreader you run the risk that accesskeys will work in one browser and not in the other. So e.g. <code>input tabindex=-1 accesskey=e</code> can be focused in Firefox with the keyboard, but not in IE 11, Chrome and Edge Chromium, because in these three browsers with Alt+E a browser-specific function is called out. This would not be a violation of the new SC, but a problem of the browser. But it shows how important it is to support standard operation and not to rely on accesskeys, which might not work at all.
* Statistically, 2.1.1 (besides 1.1.1, 1.3.1 and 4.1.2) has the most problems with accessibility tests. Therefore, it would be good to close this loophole at 2.1.1.
* The problem affects many user groups. Primarily blind and motor-impaired users. But also cognitively and visually impaired people sometimes work with the keyboard.

Impact
* Strong improvement of keyboard usability in few cases where it is violated.
* Affects only a few pages and applications, as problems often occur with 2.1.1, but the new criterion would rarely be violated.
* The developers could either implement the standard operation (high effort) or simply describe its operation (very low effort).

For WCAG 3.0, it would make sense to combine 2.1.1 and the new criterion, as was done for 2.1.2.

I suggest AA as the level. 

During the test I very rarely encounter keyboard traps, but very often hard to use web applications, because the standard keyboard operation was not implemented and the divergences were not documented. That's why I think this is an important addition.

If necessary, it could be considered to make the new criterion more general and to extend it to standard operation for pointing devices, touch, etc.

**Comment by GreggVan** [comment]
In WCAG3  we have an SC that says that you must use std kbd nav techniques or document what you use so user sees before encountering them.  But this requires that we have a list of what constitutes Standard keyboard commands.  We are doing this but for 2.2 no such list exists so it would be ambiguous exactly when this applies or not
